dataplane/forwarding/protocol/packet/ip4_test.go

```diff
@@ -31,55 +31,46 @@ import (
 var ip4 = []byte{0x45, 0x01, 0x00, 0x16, 0x00, 0x00, 0x00, 0x00, 0x08, 0xff, 0x00, 0x00, 0x01, 0x02, 0x03, 0x04, 0x0a, 0x0b, 0x0c, 0x0d, 0x00, 0x00}
```

Comment:
Change to 0x31 to test masking of last two bits.

Reply:
reverted this, the IP_POS already exists and covers this field